org-code-javabuilder/lib/build.gradle

```diff
@@ -38,8 +38,8 @@ task packageSmall(type: Zip) {
 }
 
 java {
-    sourceCompatibility = JavaVersion.VERSION_1_8
-    targetCompatibility = JavaVersion.VERSION_1_8
+    sourceCompatibility = JavaVersion.VERSION_11
```

Comment:
this change is not related to this PR but fixes some IntelliJ warnings--we should be using Java version 11 everywhere

Reply:
🎉 Thanks for updating this!